### ISSUE TITLE
Suggestions/Enhancements

### ISSUE THREAD

**Issue by Arkanyus** [TheresAFewConors/Sooty#7]
Some suggestions:

More IP reputation sources, maybe operating individually but have 'scoring' option that weights them based on total results?
- https://docs.abuseipdb.com/#introduction
- https://www.apivoid.com/api/ip-reputation/

Bitcon address reputation checker:
- https://www.bitcoinabuse.com/api-docs

Some kind of phone number reputation checker:
- https://ekata.com/solutions/identity-check/

URL Scanner tool, for browsing to a suspect site:
- https://urlscan.io/about-api/

Office Safelinks Stripper, like:
- http://www.o365atp.com/

Host scanning with Nessus or Nmap, or both?

**Comment by TheresAFewConors** [comment]
Thanks for adding some more suggestions.

- I'll take a look at AbuseIPDB again, the tool was originally using it but I removed it due to issues with consistency, although it did have a nice scoring system. 

- APIvoid unfortunately costs money, so I'm not sure how many people would want to pay for an API key, although if there's enough interest I'll add it.

- Bitcoin rep checker could be useful

- I personally don't get cases where I've had to use a phone number checker often, although I wouldn't be against adding it.

- URLScan would be useful, I haven't looked at their API yet though. But if possible it will definitely be added.

- Safelinks stripper would be a nice addition to the PP decoder - I'll take a look at this one in the future.

- Im not sure how useful scanning a host would be for SOC analysts, although I have a few scripts for ReadTeam and IR based work, so I might add it to one of those tools when I get around to publishing them.

Thanks again for adding some more suggestions. I'll go through and add what I can and close this case when they've been added.
